### DISCUSSION TITLE
How to add Buttons, Radio Buttons and Checkbox to Canvas?

### DISCUSSION THREAD

**Discussion by rohitkanojiya** [fabricjs/fabric.js#7952] [Q&A] [answered]
Hello, I am trying to build some activity with fabric canvas, where i need buttons, radio buttons and checkbox which can be use for Multiple Choice Type Questions. If anyone has any idea please let me know, Thanks in advance.

**Comment by ShaMan123** [comment, marked as answer]
http://fabricjs.com/demos/
http://fabricjs.com/interaction-with-objects-outside-canvas

**Reply by rohitkanojiya** [reply]
I found one util method. Can I use this method? http://fabricjs.com/docs/fabric.util.html#.makeElement 
Let me know if there is any problem with it or you dont recommend.

**Reply by ShaMan123** [reply]
You can do what you want.
If you don't understand what it does then don't.
It's for node.
